images/build/Dockerfiles/full.Dockerfile

```diff
@@ -21,7 +21,8 @@ RUN ./build.sh golang     /opt/startupcmdgen/golang
 FROM buildpack-deps:${DEBIAN_FLAVOR}-curl
 ARG DEBIAN_FLAVOR
 ARG SDK_STORAGE_BASE_URL_VALUE="https://oryx-cdn.microsoft.io"
-ARG AI_KEY
+ARG AI_CONNECTION_STRING
```

Comment:
I couldn't find any preexisting reference to `AI_CONNECTION_STRING` or `ORYX_AI_CONNECTION_STRING`. Could you please clarify why we wouldn't directly call the arg ORYX_AI_CONNECTION_STRING? Is it more consistent with the rest of similar arguments?

Reply:
Used `ORYX_AI_CONNECTION_STRING` and `ORYX_AI_CONNECTION_STRING` in the same manner `ORYX_AI_INSTRUMENTATION_KEY` and `AI_KEY` are used throughout the repo now just to be consistent.